trade-finance-manager-api/README.md

```diff
@@ -48,7 +47,6 @@ For example, when making a single deal query in the GraphQL playground, use the
 
 - HTTP headers:
 
-```shell
```

Comment:
I think we removed this line by accident

Reply:
Thanks, have added back.